_posts/2021-10-06-what-is-seraphis.md

```diff
@@ -0,0 +1,59 @@
+
+Seraphis is, similarly, a fourth-generation privacy-focused transaction protocol (abstraction). Also like Triptych, it defines a new key image construction that permits efficient one-of-many proofs. However, there are a number of notable differences between Triptych and Seraphis.
+
+
+### Triptych vs. Seraphis
+
+#### Triptych
+- Pros
+    - Compatible with CryptoNote user @addresses.
+- Cons
+    - Multisig is [much harder](https://github.com/cypherstack/triptych-multisig) with Triptych-style key images than with either CryptoNote or Seraphis key images.
+
+#### Seraphis
+
+When thinking about a possible implementation...
+
+- Pros
+    - Multisig can be simple, comparable to multisig with MLSAG/CLSAG.
+    - 'Membership proof delegation' is possible without leaking users' private keys. This potentially allows the following:
+        - Transaction chaining (required for full-featured atomic swaps).
+        - Offload membership proof construction to a third party, or a view-only @wallet (possibly useful for spend-only hardware wallets where it is expensive/difficult to implement complex algorithms).
+        - Ignore 10-block lock time when transacting with a **trusted party** (i.e. allow them to make your tx's membership proofs and submit the tx to the network on your behalf).
+    - Flexible multi-tier address schemes. The following features can be designed into a Seraphis user address scheme (there are at least 7 design variations, half of which require 3-key addresses [a 50% increase in address length over current addresses]):
+        - View-only wallet that can see spent outputs. Especially helpful for multisig and hardware wallets, since in RingCT and CryptoNote it is necessary to constantly 'export key images'.
+        - View-only wallet that can see received outputs but not amounts (and optionally also see spent outputs but not amounts). Useful for delegated chain scanning, where a third party or insecure machine scans for outputs, then sends them to a more secure 2nd tier view-only wallet (or spend wallet) that can read the amounts.
+        - View-only wallet that can detect the Janus attack.
+    - Modular core design. This will hopefully make it easier to upgrade different pieces of a Seraphis implementation as innovations appear (or maybe we will be stuck for the next 20 years... who knows?).
+- Cons
+    - Not compatible with CryptoNote user addresses. If Seraphis is used to upgrade a cryptocurrency that has CryptoNote-style addresses, then users would have to replace all their public CryptoNote-style addresses with Seraphis-style addresses (they would not need new private keys or wallets).
+
+
+### What about Lelantus-Spark?
```

Comment:
Generally, I'd like to see this section more fleshed out. I still have no meaningful idea why one over the other.

Reply:
Like I said, they are basically equivalent. There is nothing more to say, except - I already implemented a PoC of Seraphis so why would I make a whole new PoC for something with no functional differences?